frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -266,7 +274,7 @@ export const InventoryLayer: React.FC<InventoryLayerProps> = ({
           // Fit to municipality bounds
           map.fitBounds((GeoJSON.geometryToLayer(municipality) as any)._bounds, { maxZoom: 11 });
         }
-      } else if (results.length > 0) {
+      } else if (results.length > 0 && isMapView && isSidebarOpen === 'false') {
```

Comment:
I'm not sure if there's a more elegant way of doing this, such as preventing the search from getting called...

Reply:
I see, is this also the same issue in production?